packages/car/src/index.ts

```diff
@@ -121,22 +121,27 @@ export interface Car {
    * ```typescript
    * import fs from 'node:fs'
    * import { Readable } from 'stream'
+   * import { car } from '@helia/car'
+   * import { CarWriter } from '@ipld/car'
    * import { createHelia } from 'helia'
-   * import { car } from '@helia/car
    * import { CID } from 'multiformats/cid'
-   * import pEvent from 'p-event'
    *
    * const helia = await createHelia()
    * const cid = CID.parse('QmFoo...')
    *
    * const c = car(helia)
-   *
-   * const byteStream = await c.export(cid)
+   * const { writer, out } = CarWriter.create(cid)
+   * c.export(cid, writer)
```

Comment:
Prevent unhandled rejections:

```suggestion
   * c.export(cid, writer).catch(console.error)
```